hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -1962,6 +1962,11 @@ public boolean waitForFlushes(long timeout) {
     }
   }
 
+  @Override
+  public Configuration getConfiguration() {
```

Comment:
To be used in CPs?  Should we create a new Configuration(this.conf)?

Reply:
We have a ReadOnlyConfiguration. @mymeiyi 

org.apache.hadoop.hbase.coprocessor.ReadOnlyConfiguration